ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk

```diff
@@ -1,28 +1,28 @@
-back = esc
-start = enter
-a = space
-b = b
-x = x
-y = y
-l1 = mouse_right
-l2 = n
-l3 = g
-r1 = mouse_left
-r2 = m
-r3 = h
-guide = h
-up = up
-down = down
-left = left
-right = right
-left_analog_up = w
-left_analog_down = s
-left_analog_left = a
-left_analog_right = d
-right_analog_up = mouse_movement_up
-right_analog_down = mouse_movement_down
-right_analog_left = mouse_movement_left
-right_analog_right = mouse_movement_right
+back =  n                                       # Quick Load
+start = m                                       # Quick Save
+guide = esc                                     # Pause / Menu  
```

Comment:
As we talked about most devices don't have a guide button so we need to add a alternative combo to get access to the esc button. 

Hotkey information can be found here

https://github.com/PortsMaster/gptokeyb#hotkey--button-for-additional-key-assignments

Hotkey by default is the back button + the hotkey you assigned.

Reply:
`esc` should be assigned to `back`, and let the user reassign as needed. This is how it works in Harkinian ports.

Reply:
yeah I just don't care about this anymore, removing the PR